app/views/languages/_repo.erb

```diff
@@ -0,0 +1,18 @@
+<% if !track.active %>
+<div class="track_link col-xs-4 col-sm-4 col-md-2 col-lg-2 text-center" >
+<a href="<%= track.repository%>">
+ <p> <%= track_icon(track.id) %> </p>
+ <p><%= track.language %></p>
+</a>
+<p><a href="https://github.com/exercism/x<%= track.id %>/issues/<%= track.checklist_issue %>">Launch checklist</a> </p>
+</div>
+<% end %>
+
+<% if track.active %>
```

Comment:
Inactive tracks are in a div, whereas active tracks are not. Could we make them more similar so that we can combine some of this code?